content/dips/DIP-17.md

```diff
@@ -0,0 +1,53 @@
+---
+DIP: 17
+title: Winding Down the Safety Staking Module
+status: Proposed
+author: Xenophon Labs
+shortDescription: Wind down the existing safety staking module
+discussions: https://forums.dydx.community/discussion/7013-dydx-safety-staking-review
+created: 2022-11-02
+---
+
+## Simple Summary
+
+The dYdX Safety Staking Module, which allowed users to stake DYDX that would then be used as an insurance backstop in the case of a protocol shortfall event, will be shut down. The remaining rewards allocated to the Safety Staking Module will not be distributed, but accrue in the Rewards Treasury, which in turn can be directed by the community.
+
+## Abstract
+
+Xenophon Labs proposed winding down the Safety Staking Module due to its impracticality as an insurance fund and inefficient allocation of DYDX token.
+
+Each epoch, the 383,562 DYDX currently allocated to DYDX stakers will instead accummulate in the rewards treasury. The dYdX community may redirect the DYDX from the rewards treasury with a vote falling under the requirement of the short timelock. Also, an additional proposal can be submitted to transfer the DYDX back to the rewards treasury if needed.
```

Comment:
Each epoch, the 383,562 DYDX currently allocated to DYDX stakers will instead accummulate in the rewards treasury. The dYdX community may redirect the DYDX from the rewards treasury with a vote falling under the requirement of the short timelock. Also, an additional proposal can be submitted to transfer the DYDX back to the [**rewards treasury**--> Community Treasury] if needed.